discovery.py

```diff
@@ -35,6 +35,12 @@ def query(self, queryString, label):
                                         qopts)
 
         matches = my_query['results']
+        htmlList = []
+        max_score = matches[0]['score']
+        for i in range(3):
+            if (max_score - matches[i]['score'] <0.1):
```

Comment:
can you fix the spacing with the "<0.1" to "< 0.1" It will fuck with our linter.